### PR TITLE
Add Cleartext.app v1.3.1

### PR DESCRIPTION
**Pull request by michaelbanfield**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/cleartext.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'cleartext' do
+  version '1.3.1'
+  sha256 'c53860370de86d59d81eb1164df1ea17a7824333be4eb2beb52843663dbb5b27'
+
+  url 'https://github.com/mortenjust/cleartext-mac/releases/download/1.31/Cleartext1.31.zip'
+  name 'Cleartext'
+  homepage 'https://github.com/mortenjust/cleartext-mac'
+  license :mit # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/cleartext.rb]: Remove the comment (like it says).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1cabcbbf6ac6b2c1ea23920cbc388d108b11c0e3. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by johnliu** [comment]
@vitorgalvao your updates to the PR broke this. It the version should have been `1.31` instead of `1.3.1`, changing the url to interpolate the version causes an error.

**Comment by vitorgalvao** [comment]
Will fix shortly.

**Pull request closed without merging**